.ci/packaging.groovy

```diff
@@ -429,13 +429,11 @@ def triggerE2ETests(String suite) {
     booleanParam(name: 'notifyOnGreenBuilds', value: !isPR()),
     booleanParam(name: 'BEATS_USE_CI_SNAPSHOTS', value: true),
     string(name: 'runTestsSuites', value: suite),
+    string(name: 'BEAT_VERSION', value: env.BEAT_VERSION),
     string(name: 'GITHUB_CHECK_NAME', value: env.GITHUB_CHECK_E2E_TESTS_NAME),
     string(name: 'GITHUB_CHECK_REPO', value: env.REPO),
     string(name: 'GITHUB_CHECK_SHA1', value: env.GIT_BASE_COMMIT),
   ]
-  if (isPR()) {
```

Comment:
I think this change is unintencional.

Reply:
forget it is the same code it is in the master, I dunno why this appears as a change

https://github.com/elastic/beats/blob/master/.ci/packaging.groovy#L420-L442

Reply:
same here, but it is fixed now